include/contin_array.h

```diff
@@ -9,20 +9,30 @@
 
 #include <zephyr/kernel.h>
 
-/** @brief. Creates a continuous array from a finite array
- * @param pcm_cont		Pointer to the destination array
+/**
+ * @file contin_array.h
+ *
+ * @defgroup contin_array Continuous array
+ * @{
+ * @brief Basic continuous array.
+ */
+
+/** @brief Creates a continuous array from a finite array.
+ *
+ * @param pcm_cont		Pointer to the destination array.
  * @param pcm_cont_size	        Size of pcm_cont.
- * @param pcm_finite		Pointer to an array of samples/data
- * @param pcm_finite_size	Size of pcm_finite
+ * @param pcm_finite		Pointer to an array of samples or data.
+ * @param pcm_finite_size	Size of pcm_finite.
  * @param finite_pos		Variable used interally. Must be set
- *				to 0 for first run and not changed.
+ *				to 0 for the first run and not changed.
  *
  * @note  This function serves the purpose of e.g. having a set of audio samples
  * stored in pcm_finite. This can then be fetched in smaller pieces into ram and
  * played back in a loop using the results in pcm_cont.
  *
- * @return 0 on success.        -EPERM if any sizes are zero
- *				-ENXIO on NULL pointer.
+ * @retval 0		If the operation was successful.
+ * @retval -EPERM 	If any sizes are zero.
+ * @retval -ENXIO	On NULL pointer.
  */
 int contin_array_create(void *pcm_cont, uint32_t pcm_cont_size, void const *const pcm_finite,
 			uint32_t pcm_finite_size, uint32_t *const finite_pos);
```

Comment:
Please add the following lines before the closing #endif to solve the doc build issue:
```
/**
 * @}
 */
```